spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkUtil.java

```diff
@@ -179,4 +191,63 @@ public static Configuration hadoopConfCatalogOverrides(SparkSession spark, Strin
   private static String hadoopConfPrefixForCatalog(String catalogName) {
     return String.format(SPARK_CATALOG_HADOOP_CONF_OVERRIDE_FMT_STR, catalogName);
   }
+
+  /**
+   * Get a List of Spark filter Expression.
+   *
+   * @param schema table schema
+   * @param filters filters in the format of a Map, where key is one of the table column name,
+   *                and value is the specific value to be filtered on the column.
+   * @return a List of filters in the format of Spark Expression.
+   */
+  public static List getSparkFilterExpressions(StructType schema,
```

Comment:
Iceberg doesn't use `get` in method names because it tends to either be filler or prevent us from having more specific methods. Here, I think a more specific name is `partitionMapToExpression`

Reply:
Fixed. Thanks!